src/components/dataset/list.vue

```diff
@@ -10,10 +10,17 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <div>
-    <dataset-introduction/>
-    <dataset-table/>
-    <loading :state="datasets.initiallyLoading"/>
+  <div id="dataset-list-page">
+    <page-section>
+      <template #heading>
+        <h1>Datasets</h1>
```

Comment:
The `PageSection` component actually expects this to be a `<span>`. `PageSection` has styles to set the color and font size. Maybe we should change `PageSection` though so that it works with `<h*>` elements: using those elements makes a lot of sense.